partner_email_check/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Email Format Checker",
+    "name": "Partner Email Format and Duplicate Checker",
     "version": "14.0.1.1.1",
     "summary": "Validate email address field",
```

Comment:
"summary": "Normalize, validate and/or prevent duplicates in email address field,",